tests/SpecTests/ClientSideEncryption/Prose22_RangeExplicitEncryptionTest.php

```diff
@@ -1,463 +0,0 @@
-<?php
-
-namespace MongoDB\Tests\SpecTests\ClientSideEncryption;
-
-use ArrayIterator;
-use Generator;
-use Iterator;
-use MongoDB\BSON\Binary;
-use MongoDB\BSON\Decimal128;
-use MongoDB\BSON\Document;
-use MongoDB\BSON\Int64;
-use MongoDB\BSON\UTCDateTime;
-use MongoDB\Client;
-use MongoDB\Collection;
-use MongoDB\Driver\ClientEncryption;
-use MongoDB\Driver\Exception\EncryptionException;
-use MultipleIterator;
-
-use function base64_decode;
-use function file_get_contents;
-use function get_debug_type;
-use function is_int;
-use function phpversion;
-use function version_compare;
-
-/**
- * Prose test 22: Range Explicit Encryption
- *
- * @see https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.md#22-range-explicit-encryption
- * @group csfle
- * @group serverless
- */
-class Prose22_RangeExplicitEncryptionTest extends FunctionalTestCase
-{
-    private ?ClientEncryption $clientEncryption = null;
-    private ?Collection $collection = null;
-    private ?Client $encryptedClient = null;
-    private $key1Id;
-
-    public function setUp(): void
-    {
-        parent::setUp();
-
-        if (version_compare(phpversion('mongodb'), '1.20.0dev', '>=')) {
-            $this->markTestIncomplete('Range protocol V1 is not supported by ext-mongodb 1.20+');
-        }
```

Comment:
This should still be a valid prose test. I think the problem is that I didn't remove this skip when adding the server 8.0+ requirement below in https://github.com/mongodb/mongo-php-library/commit/0b16223727cd2a0d92d12b71035c03dbc9e53558.

Based on the [prose test definition](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.md#22-range-explicit-encryption), it should be run for server 8.0+ and libmongocrypt 1.11+. The accompanying JSON files have already been updated for QEv2 (since https://github.com/mongodb/specifications/commit/9bfa512e41c1231a51f83260f5672a36626d9bcd).

Reply:
See: https://github.com/mongodb/mongo-php-library/pull/1400